modules/apps/list-type/list-type-test/src/testIntegration/java/com/liferay/list/type/service/test/ListTypeDefinitionLocalServiceTest.java

```diff
@@ -57,13 +58,11 @@ public class ListTypeDefinitionLocalServiceTest {
 
 	@Test
 	public void testAddListTypeDefinition() throws Exception {
-		ListTypeDefinition listTypeDefinition =
-			_listTypeDefinitionLocalService.addListTypeDefinition(
-				TestPropsValues.getUserId(),
-				Collections.singletonMap(
-					LocaleUtil.US, RandomTestUtil.randomString()));
+		ListTypeDefinition listTypeDefinition = _addListTypeDefinition();
 
 		Assert.assertNotNull(listTypeDefinition);
+		Assert.assertNotNull(listTypeDefinition.getExternalReferenceCode());
+		Assert.assertNotNull(listTypeDefinition.getName());
```

Comment:
@murilostodolni `assertNotNull` only checks if it is not `null`? Wouldn't it be better to check that it's not empty?

```
Assert.assertTrue(Validator.isNotNull(listTypeDefinition.getExternalReferenceCode()));
Assert.assertTrue(Validator.isNotNull(llistTypeDefinition.getName()));
```